client/my-sites/plugins/main.jsx

```diff
@@ -344,6 +367,23 @@ const PluginsMain = React.createClass( {
 		} );
 	},
 
+	handleAddPluginButtonClick() {
+		this.props.recordGoogleEvent( 'Plugins', 'Clicked Add New Plugins' );
+	},
+
+	getAddPluginButton() {
+		const browserUrl = '/plugins/browse' + ( this.props.selectedSiteSlug ? '/' + this.props.selectedSiteSlug : '' );
+		return (
+			<HeaderButton
+				icon="plus"
+				label={ this.props.translate( 'Add Plugin' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'Add a Plugin' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).